packages/firebase_ml_vision/lib/src/text_recognizer.dart

```diff
@@ -63,6 +82,36 @@ class TextRecognizer {
   }
 }
 
+/// Options for cloud text recognizer.
```

Comment:
nit: I believe this should be a little bit more descriptive: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-useless-documentation